conda_rpms/tests/unit/build_rpm_structure/test_create_rpmbuild_for_tag.py

```diff
@@ -0,0 +1,65 @@
+from mock import patch, MagicMock
+import os
+
+import conda_rpms.tests as tests
+from conda_rpms.build_rpm_structure import create_rpmbuild_for_tag
+
+
+CONFIG = {'rpm' : {'prefix' : 'SciTools'},
+          'install' : {'prefix' : 'test_install_location'}}
+
+ENV_SPEC = """channels:
+ - https://conda.anaconda.org/conda-forge/linux-64
+env:
+ - python
+"""
+
+ENV_MANIFEST ="""https://conda.anaconda.org/conda-forge/linux-64\tca-certificates-2018.1.18-0
+https://conda.anaconda.org/conda-forge/linux-64\tncurses-5.9-10
+https://conda.anaconda.org/conda-forge/linux-64\topenssl-1.0.2n-0
+https://conda.anaconda.org/conda-forge/linux-64\tpython-3.6.4-0
+https://conda.anaconda.org/conda-forge/linux-64\tsqlite-3.20.1-2
+https://conda.anaconda.org/conda-forge/linux-64\ttk-8.6.7-0
+https://conda.anaconda.org/conda-forge/linux-64\txz-5.2.3-0
+https://conda.anaconda.org/conda-forge/linux-64\tzlib-1.2.11-0
+"""
+
+
+class Test(tests.CommonTest):
+    @patch('conda_rpms.build_rpm_structure.create_rpmbuild_for_env')
+    def test_sorted_order(self, a):
+        with self.temp_dir() as target:
+            with self.temp_dir() as temp_repo:
+                # Create fake env.spec and manifest files
+                with open(os.path.join(temp_repo + '/env.spec'), 'w') as \
+                        env_spec_file:
+                    env_spec_file.write(ENV_SPEC)
+                with open(os.path.join(temp_repo + '/env.manifest'), 'w') as \
```

Comment:
@lbdreyer Note that you don't need the leading `\` as `os.path.join` will do this for you.